src/modules/loadevents.js

```diff
@@ -0,0 +1,13 @@
+import * as myEvents from './eventListeners.js';
```

Comment:
- [ ] eventListeners.js file is currently empty. You will require a checkbox and an event listener on this checkbox. Please let's implement this. 
- [ ] Also, from the requirements of this project, the task object's values for `completed` are updated upon user actions( ie clicking on the checkbox)